src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  line-height: 60px;
+  background: white;
+}
+
+.nav__list {
```

Comment:
Avoid using tag names for styling. Use class selectors instead to ensure that the styles are not overly broad and can be reused efficiently.